src/lib.rs

```diff
@@ -113,6 +196,20 @@ fn create_git(args: Args, name: &ProjectName) -> Result<()> {
     Ok(())
 }
 
+fn get_config_file_values(path: &Path) -> Result<HashMap<String, toml::Value>> {
+    match fs::read_to_string(path) {
+        Ok(ref contents) => toml::from_str::<ConfigValues>(contents)
+            .map(|v| v.values)
+            .map_err(|e| e.into()),
+        Err(e) => anyhow::bail!(
+            "{} {} {}",
+            emoji::ERROR,
+            style("VAlues File Error:").bold().red(),
```

Comment:
here is a little typo, `VAlues` but should be `Value File Error` possibly

Reply:
Done